src/testsuite/arith128_print.c

```diff
@@ -1365,7 +1365,7 @@ check_int64 (char *prefix, long val64, long shouldbe)
   return (rc);
 }
 
-#ifndef __clang__ // Clang does not support _Decimal* types
+#ifndef PVECLIB_DISABLE_DFP // Clang does not support _Decimal* types
```

Comment:
delete the comment at the end of this line?

Reply:
OK